### PR TITLE
New jquery version added

### PR DESCRIPTION
**Pull request by mohit-003**
New jQuery version-3.4.1 added and replaced

### CODE REVIEW

**Comment by mohit-003** [comment]
@sal2701 Changed it

**Comment by mohit-003** [comment]
From https://jquery.com/download/

File: _includes/head.html [hunk 1]
```diff
@@ -31,8 +31,8 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   
-  <script defer src="\assets\js\vendor\fontawesome-solid.js"></script>
-  <script defer src="assets\js\vendor\fontawesome-main.js"></script>
+  <script defer src="../assets/js/vendor/fontawesome-solid.js"></script>
```
[the remark this answers is not in the document]
Comment (mohit-003) [reply _includes/head.html]: Even i observed it just now

**Comment by sal2701** [comment]
@tachyons @satu0king @ankitkataria please review. Works as expected.

**Comment by mohit-003** [comment]
I did a mistake and changed the files

**Pull request closed without merging**